### ISSUE TITLE
Option to make pip window unclickable/uninteractable

### ISSUE THREAD

**Issue by Jonas164** [GoogleChromeLabs/picture-in-picture-chrome-extension#58]
Sometimes it would be convenient if clicking into the pip window would be ignored.

**Comment by Derkness** [comment]
This would be a very good feature for watching videos while playing games.

**Comment by Spuner** [comment]
It would be nice if we added such a feature.

**Comment by eduardoftdo** [comment]
Is there any way to do this at the moment? It would be a nice feature to have

**Comment by Stroichik** [comment]
Yeah, PiP lock would a nice feature. No one has at the moment, at least i don't know any app that has this feature

**Comment by drachescript** [comment]
would love that myself but sadly a extention cant do that for now. we cant control the popup on how it reacts on input.


there are workarounds how to make a window unclickable, i heard, but no idea about that

**Comment by Spuner** [comment]
There is a solution to make the PiP window inactive altogether, but this is done through AutoHotKey.

**Comment by drachescript** [comment]
> There is a solution to make the PiP window inactive altogether, but this is done through AutoHotKey.

yep thats what i meant with workarounds
